content/zh/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md

```diff
@@ -1,106 +1,109 @@
+- 您需要创建一个创建企业空间，一个项目 (`demo-project`) 和一个帐户 (`project-regular`)。`project-regular` 需要被邀请到项目中，并赋予 `operator` 角色。有关详细信息，请参考[创建企业空间、项目、帐户和角色](../../../quick-start/create-workspace-and-project/#step-1-create-an-account)。
+- 您需要创建一个 S2I 镜像构建器和 B2I 镜像构建器。有关更多信息，请参见 [Source to Image：无需 Dockerfile 发布应用](../source-to-image/)和[Binary to Image：发布制品到 Kubernetes](../binary-to-image/)。
 
-## Configure an S2I Webhook
+## 配置 S2I Webhook
 
-### Step 1: Expose the S2I trigger Service
+### 步骤 1: 暴露 S2I trigger 服务
 
-1. Log in to the KubeSphere web console as `admin`. Click **Platform** in the top-left corner and then select **Cluster Management**.
+1. 以 `admin` 身份登录 KubeSphere Web 控制台。在左上角点击**平台管理**，然后选择**集群管理**。
 
-2. In **Services** under **Application Workloads**, select **kubesphere-devops-system** from the drop-down list and click **s2ioperator-trigger-service** to go to its detail page.
+2. 选择在**应用负载**下面的**服务**，从下拉框中选择 **kubesphere-devops-system**，然后点击 **s2ioperator-trigger-service** 进入详情页面。
 
    ![s2i-trigger-service](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/s2i-trigger-service.png)
 
-3. Click **More** and select **Edit Internet Access**.
+3. 点击**更多操作**，选择**编辑外网访问**。
 
    ![edit-trigger-service](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/edit-trigger-service.png)
 
-4. In the window that appears, select **NodePort** from the drop-down list for **Access Method** and then click **OK**.
+4. 在出现的窗口中，从**访问方法**的下拉菜单中选择 **NodePort**，然后点击**确定**。
 
    ![select-nodeport](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/select-nodeport.png)
 
    {{< notice note >}}
 
-   This tutorial selects **NodePort** for demonstration purposes. You can also select **LoadBalancer** based on your needs.
+   本教程出于演示目的选择 **NodePort**。根据您的需要，您也可以选择 **LoadBalancer**。
 
    {{</ notice >}}
 
-5. You can view the **Node Port** on the detail page. It will be included in the S2I webhook URL.
+5. 在详情界面可以查看 **Node Port**。Node Port 会包括在 S2I webhook URL 中。
 
    ![s2i-nodeport](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/s2i-nodeport.png)
 
-### Step 2: Configure an S2I webhook
+### 步骤 2: Configure an S2I webhook
 
-1. Log out of KubeSphere and log back in as `project-regular`. Go to `demo-project`.
+1. 登出 KubeSphere 并以 `project-regular` 帐户登回。然后转到 `demo-project`。
 
-2. In **Image Builder**, click the S2I Image Builder to go to its detail page.
+2. 在**镜像构建器**中，点击 S2I 镜像构建器，进入详情页面。
 
    ![click-s2i](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/click-s2i.png)
 
-3. You can see an auto-generated link shown in **Remote Trigger Link**. Copy `/s2itrigger/v1alpha1/general/namespaces/demo-project/s2ibuilders/felixnoo-s2i-sample-latest-zhd/` as it will be included in the S2I webhook URL.
+3. 您可以在**远程触发链接**中看到自动生成的链接。复制   `c/s2itrigger/v1alpha1/general/namespaces/demo-project/s2ibuilders/yangmaozhang-s2i-sample-latest-czf/`，因为这个链接将包含在 S2I webhook URL 中。
```

Comment:
`c/s2itrigger/v1alpha1/general/namespaces/demo-project/s2ibuilders/yangmaozhang-s2i-sample-latest-czf/`
->
`/s2itrigger/v1alpha1/general/namespaces/demo-project/s2ibuilders/felixnoo-s2i-sample-latest-zhd/`

Let's use the URL from the screenshot.